modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/config/io.scala

```diff
@@ -622,4 +622,19 @@ object io {
 
     implicit val encoder: Encoder[Cloud] = Encoder.encodeString.contramap[Cloud](_.toString().toUpperCase())
   }
+
+  case class License(
+    accept: Boolean
+  )
+
+  object License {
+    implicit val licenseDecoder: Decoder[License] = {
```

Comment:
Because I find it fun to see if there's alternative ways to write the same thing in a different way.  Here's what I came up with:

```scala
    implicit val licenseDecoder: Decoder[License] = {
      val truthy = Set("true", "yes", "on", "1")
      Decoder[String]
        .map(_.toLowerCase)
        .map(truthy(_))
        .or(Decoder[Boolean])
        .map(License(_))
        .at("accept")
    }
```
or

```scala
    implicit val licenseDecoder: Decoder[License] = {
      val truthy = Set("true", "yes", "on", "1")
      Decoder[String]
        .either(Decoder[Boolean])
        .at("accept")
        .map {
          case Left(s) => License(truthy(s.toLowerCase))
          case Right(b) => License(b)
        }
    }
```

or what you did, which is completely fine and is the same as the collector